injective/assetlist.json

```diff
@@ -1068,6 +1068,31 @@
       "type_asset": "cw20",
       "address": "inj19vy83ne9tzta2yqynj8yg7dq9ghca6yqn9hyej",
       "base": "cw20:inj19vy83ne9tzta2yqynj8yg7dq9ghca6yqn9hyej",
+      "name": "cw20 DRUGS",
```

Comment:
Is this a legacy mintage of DRUGS? I.e., is this created by the same team, but being replace with this new one?

Reply:
Yes kind of. The old cw20 dev is stepping aside and a new team is taking over and minting the factory version. cw20 holders can swap for the new token 1:1 :)

Reply:
In that case I would that the old 'cw20' version be indicated as a legacy mintage using traces: [ { type: legacy-mintage} ]